packages/perseus/src/widgets/categorizer/categorizer.util.test.ts

```diff
@@ -0,0 +1,32 @@
+import splitCategorizerWidgetOptions from "./categorizer.util";
+
+import type {PerseusCategorizerWidgetOptions} from "@khanacademy/perseus-core";
+
+describe("splitCategorizerWidgetOptions", () => {
+    it("returns an object without the answer data", () => {
+        const categorizerTestWidgetOptions: PerseusCategorizerWidgetOptions = {
+            values: [0, 1],
+            items: ["apples", "oranges"],
+            categories: ["citrus", "non-citrus"],
+            randomizeItems: true,
+            static: false,
+            highlightLint: false,
+            linterContext: {
+                contentType: "type",
+                paths: ["paths"],
+                stack: ["stack"],
+            },
+        };
+
+        const splitOptions = splitCategorizerWidgetOptions(
+            categorizerTestWidgetOptions,
+        );
+
+        expect(splitOptions).toEqual({
```

Comment:
I can't think of more tests to write here. We just want to make sure that the private data is not in here, and `.toEqual()` does that.